cs-utilities/src/main/java/io/cloudslang/content/utilities/entities/constants/LocalPingDescriptions.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * (c) Copyright 2017 EntIT Software LLC, a Micro Focus company, L.P.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Apache License v2.0 which accompany this distribution.
+ *
+ * The Apache License is available at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.cloudslang.content.utilities.entities.constants;
+
+public class LocalPingDescriptions {
+
+    public static class OperationDescription {
+        public static final String LOCAL_PING_OPERATION_DESC = "This operation runs a Ping command locally.";
+    }
+
+    public static class InputsDescription {
+        public static final String IP_VERSION_INPUT_DESC = "IP version forced to the ping command executed on the target host. " +
+                "For Windows -4 or -6 parameters will be added.On Linux will be used ping or ping6. " +
```

Comment:
"On Linux will be used ping or ping6" -> "On Linux, ping or ping6 will be used" 😆